### DISCUSSION TITLE
Need to add stable Taiko judgement to lazer classic mode

### DISCUSSION THREAD

**Discussion by Sitoria** [ppy/osu#17715] [Ideas]
As i upload this in my [Youtube](https://youtu.be/9H4QujO5Zws), Taiko's Slider Judgement is different from stable. 

osu! Stable: Slider is just additional score object (not impact to acc)
osu! Lazer: Same judgement as V2 (slider + slidertick impact to acc)

i think this might be problem to convert taiko users, if they didn't hit all sliderticks, it might be accuracy down and get 100, 50 or misscount.

and, stable hit judgement hasn't been transplanted to lazer, Classic Mode. 
Stable
![osu_taiko_lazer_Slider_Judgement_Problem](https://user-images.githubusercontent.com/85770829/162337634-fa9c6487-fe57-4bfe-a045-99cb0e9a6e29.gif)

Lazer
![osu_taiko_lazer_Slider_Judgement_Problem (1)](https://user-images.githubusercontent.com/85770829/162337824-50ccdaf2-b914-4540-8b6e-853b0999b82c.gif)

Regards.

**Comment by peppy** [comment]
Need to confirm that the lazer behaviour is actually what we want going forward, as well. I'm not sure much consideration has been given to this yet.

**Reply by Sitoria** [reply]
but Unlike the community vote, this must be implemented in Taiko Mode: Classic, right? because this ruleset is stable, and lazer's classic mode is implented the stable rulesets.

**Reply by peppy** [reply]
What I was trying to say is that there's a chance we want to change the behaviour in lazer, and therefore not *require* the classic mod change itself. Would need to hear from more taiko players as to whether the new judgement style is an improvement or not.
